src/main/java/org/gridsuite/shortcircuit/server/service/ShortCircuitWorkerService.java

```diff
@@ -193,7 +193,8 @@ public void postRun(ShortCircuitRunContext runContext, AtomicReference<ReportNod
     protected void handleNonCancellationException(AbstractResultContext<ShortCircuitRunContext> resultContext, Exception exception, AtomicReference<ReportNode> rootReporter) {
         if (exception instanceof ShortCircuitException shortCircuitException && shortCircuitException.getType() == INCONSISTENT_VOLTAGE_LEVELS) {
             postRun(resultContext.getRunContext(), rootReporter, null);
-            sendResultMessage(resultContext, null);
+            notificationService.sendResultMessage(resultContext.getResultUuid(), resultContext.getRunContext().getReceiver(),
```

Comment:
I don't get the point of this change. Does the HEADER_BUS_ID in `sendResultMessage` causes a problem ?

Reply:
in sendResultMessage result.getFaultResults().isEmpty is null so it throws an error

Reply:
I think it throws ShortCircuitException(MISSING_EXTENSION_DATA, "Missing short-circuit extension data");
but I'll check

Reply:
Ok the problem was the null argument, it was not managed in the method and it was throwing

Reply:
Good. I prefer it that way.